contributing.md

```diff
@@ -0,0 +1,67 @@
+Contributing
+============
+
+## Issue Tracking
+
+New feature requests, changes, enhancements, non-methodology features, and bug reports can be filed as new issues in the
+[GitHub.com issue tracker](https://github.com/entralliance/entr_runtime/issues) at any time. Please be sure to fully describe the
+issue.
+
+<!---For other issues, please email the ENTR distribution list at `xx`.--->
+
+## Repository
+
+The entr_runtime repository is hosted on GitHub, and located here: https://github.com/entralliance/entr_runtime
+
+This repository is organized using a modified git-flow system. Branches are organized as follows:
+
+- master: Stable release version. Must have good test coverage and may not have all the newest features.
+- dev: Development branch which contains the newest features. Tests must pass, but code may be unstable.
+- feature/xxx: Branch from dev, should reference a GitHub issue number.
```

Comment:
I like including fix/ and hotfix/ branch names, but that may be overcomplicating things

Reply:
will add with a quick group consensus